reactNativeProject/src/screens/Main/components/TodoList/components/Todo/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import { View, StyleSheet, Text, TouchableOpacity } from "react-native";
+import PropTypes from "prop-types";
+import Checkbox from "../checkbox";
+import styles from "./styles";
+
+export default class Todo extends Component {
+  render() {
+    const { item, onToggleItemCompleted, onRemoveItem } = this.props;
+    const itemStyle = item.completed
+      ? [styles.item, styles.completed]
+      : styles.item;
+    return (
+      <View key={item.label} style={itemStyle}>
+        <Text> {item.label} </Text>
+        <View style={styles.rightSection}>
+          <Checkbox
+            isChecked={item.completed}
+            onToggle={onToggleItemCompleted}
+          />
+          <TouchableOpacity onPress={onRemoveItem}>
+            <Text style={styles.remove}> &times; </Text>
```

Comment:
You don´t need this character. You can just use the `Button` component from the `react-native` package